app/src/main/res/values/strings.xml

```diff
@@ -842,6 +842,15 @@ Before uploading your changes, the app checks with a &lt;a href=\"https://www.we
     <string name="quest_camera_type_fixed">Fixed</string>
     <string name="quest_camera_type_panning">Panning</string>
 
+    <string name="quest_camp_drinking_water_title">Is drinking water freely available to customers here?</string>
+    <string name="quest_camp_shower_title">Are there showers here?</string>
+    <string name="quest_camp_power_supply_title">Can customers charge their phone/laptop here?</string>
```

Comment:
Maybe 

> Is there any way for customers to charge their phone or laptop here?

to make more clear that even poorly accessible charging counts? (not a native speaker, not entirely sure)

Reply:
I think that it is almost ready for final review by `@westnordost` - though maybe you may want to reject/use/process this idea first.

Reply:
thanks, that sounds reasonable translation, I've updated it.